### PR TITLE
feat: remove swagger-annotations dependency

### LINKED ISSUE #295: proposal: Update Swagger dependencies to v3 (OpenApi generator v5)
Hi,
not very urgent proposal, but would be a "nice to have".

__Proposal:__
Updated internal dependencies from Swagger2 `io.swagger:swagger-annotations:1.6.1` to OpenAPI (a.k.a Swagger3) `io.swagger.core.v3:swagger-annotations:2.1.12`

__Current behavior:__
No OpenAPI integration.

__Desired behavior:__
OpenAPI integration.

__Use case:__
Writing an API with Influx.

Thank you

### PR DESCRIPTION
**Pull request by rhajek**
Closes #295 

This PR removes dependency on swagger annotations and adds new APIs:

* `ConfigService`
* `DebugService`
* `MetricsService`
* `RemoteConnectionsService`
* `ReplicationsService`

## Proposed Changes

_Briefly describe your proposed changes:_

## Checklist

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergeable
- [ ] A test has been added if appropriate
- [ ] `mvn test` completes successfully
- [ ] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [ ] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,18 @@
...
      - `io.reactivex.schedulers.Schedulers` -> `io.reactivex.rxjava3.schedulers.Schedulers`
-          
+
+### Services
+
+This release also uses new version of InfluxDB OSS API definitions - [oss.yml](https://github.com/influxdata/openapi/blob/master/contracts/oss.yml). The following breaking changes are in underlying API services and doesn't affect common apis such as - `WriteApi`, `QueryApi`, `BucketsApi`, `OrganizationsApi`...
+
+- Add `ConfigService` to retrieve InfluxDB's runtime configuration
+- Add `RemoteConnectionsService` to deal with registered remote InfluxDB connections
+- Add `MetricsService` to deal with exposed prometheus metrics
```
Comment (bednar) [inline CHANGELOG.md:30]: Please add also notes about: `DebugService` and `ReplicationService`.

**Review by bednar — CHANGES_REQUESTED** [review]

File: client/src/main/java/com/influxdb/client/TelegrafsApi.java [hunk 2]
```diff
@@ -174,11 +174,11 @@ Telegraf createTelegraf(@Nonnull final String name,
     /**
      * Create a telegraf config.
      *
-     * @param telegrafRequest Telegraf Configuration to create
+     * @param telegrafPluginRequest Telegraf Configuration to create
      * @return Telegraf config created
      */
     @Nonnull
-    Telegraf createTelegraf(@Nonnull final TelegrafRequest telegrafRequest);
+    Telegraf createTelegraf(@Nonnull final TelegrafPluginRequest telegrafPluginRequest);
```
Comment (bednar) [inline client/src/main/java/com/influxdb/client/TelegrafsApi.java:181]: This is breaking change in our public API and should be notice in CHANGELOG.md.

File: client/src/main/java/com/influxdb/client/TelegrafsApi.java [hunk 3]
```diff
@@ -217,7 +217,7 @@ Telegraf createTelegraf(@Nonnull final String name,
      */
     @Nonnull
     Telegraf updateTelegraf(@Nonnull final String telegrafID,
-                            @Nonnull final TelegrafRequest telegrafRequest);
+                            @Nonnull final TelegrafPluginRequest telegrafRequest);
```
Comment (bednar) [inline client/src/main/java/com/influxdb/client/TelegrafsApi.java:220]: This is breaking change in our public API and should be notice in CHANGELOG.md.

File: client/src/main/java/com/influxdb/client/TelegrafsApi.java [hunk 4]
```diff
@@ -57,7 +57,7 @@
  *                 .putConfigItem("urls", Collections.singletonList("http://127.0.0.1:9999"));
  *
  * TelegrafPlugin cpu = new TelegrafPlugin()
- *                 .type(TelegrafPlugin.TypeEnum.INPUTS)
+ *                 .type(TelegrafPlugin.TypeEnum.INPUT)
```
Comment (bednar) [inline client/src/main/java/com/influxdb/client/TelegrafsApi.java:60]: This is breaking change in our public API and should be notice in CHANGELOG.md.

**Review by bednar — CHANGES_REQUESTED** [review]

**Comment by rhajek** [comment]
CHANGELOG.md was updated.

**Review by bednar — APPROVED** [review]
LGTM 👍

**Pull request merged**